Formula/neo4j.rb

```diff
@@ -13,17 +13,13 @@ class Neo4j < Formula
 
   bottle :unneeded
 
-  # Upstream does not intend to provide Java 8+ support until 4.0
-  # and there are various issues with running against newer Javas.
-  # https://github.com/neo4j/neo4j/issues/11728#issuecomment-387038804
-  # https://github.com/neo4j/neo4j-browser/issues/671#issuecomment-346224754
-  # https://github.com/Homebrew/homebrew-core/issues/31090
-  # As of v4.1.0, neo4j still needs to build with java 1.8
-  # https://github.com/neo4j/neo4j/issues/12516
-  depends_on java: "1.8"
+  depends_on "openjdk@11"
 
   def install
-    ENV["NEO4J_HOME"] = libexec
+    env = {
+      JAVA_HOME:  Formula["openjdk@11"].opt_prefix,
```

Comment:
is there anyway that we can use `overridable_java_home`?

Reply:
I thought about it, but felt that there wouldn't be much need for it. It's a database, not a development tool of some kind that would actually benefit from allowing this to be overridden. See https://github.com/Homebrew/homebrew-core/pull/60128#issuecomment-679455200.

Reply:
Got you, that comment link is helpful!